umu/umu_run.py

```diff
@@ -77,6 +80,15 @@ def parse_args() -> Namespace | tuple[str, list[str]]:  # noqa: D103
         parser.print_help(sys.stderr)
         sys.exit(1)
 
+    # Show version
+    # Need to avoid clashes with later options (for example: wineboot -u)
+    #   but parse_args scans the whole command line.
+    # So look at the first argument and see if we have -v or --version
+    #   in sort of the same way parse_args would.
+    if sys.argv[1].lower() == "-v" or sys.argv[1].lower().startswith("--v"):
```

Comment:
Would reexpress as:

```
    if sys.argv[1].endswith(("--version", "-v")):
        print(
            f"umu-launcher version {__version__} ({sys.version})",
            file=sys.stderr,
        )
        sys.exit(0)
```

Reply:
Changes made and committed.
Many thanks.